### PR TITLE
Increase log levels for gRPC retry loggers to ERROR

### PR DESCRIPTION
**Pull request by kuujo**

### CODE REVIEW

File: onos-config/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: onos-config
-version: 1.6.4
+version: 1.6.5
 kubeVersion: ">=1.17.0"
 appVersion: v0.10.18
```
Comment (adibrastegarnia) [inline onos-config/Chart.yaml:5]: since you are here, you can change this to v0.10.19

File: onos-config/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: onos-config
-version: 1.6.4
+version: 1.6.5
 kubeVersion: ">=1.17.0"
-appVersion: v0.10.18
+appVersion: v0.10.19
```
Comment (adibrastegarnia) [inline onos-config/Chart.yaml:5]: image tag also?

Comment (kuujo) [reply onos-config/Chart.yaml:5]: It's done by a template

Comment (kuujo) [reply onos-config/Chart.yaml:5]: https://github.com/onosproject/onos-helm-charts/blob/3d42ffe818527da564ca3e9d68bec2c14d948051/onos-config/values.yaml#L26

**Pull request merged**